reinforcement_learning/rl_unity_ray/Dockerfile

```diff
@@ -0,0 +1,46 @@
+ARG CPU_OR_GPU
+ARG AWS_REGION
+FROM 462105765813.dkr.ecr.${AWS_REGION}.amazonaws.com/sagemaker-rl-ray-container:ray-0.8.2-tf-${CPU_OR_GPU}-py36
+
+WORKDIR /opt/ml
+
+# Unity dependencies
+
+# RUN apt-get update && apt-get install -y \
+#       git cmake ffmpeg pkg-config \
+#       qtbase5-dev libqt5opengl5-dev libassimp-dev \
+#       libtinyxml-dev \
+#       libgl1-mesa-dev \
+#     && cd /opt \
+#     && apt-get clean && rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/*
+
+RUN apt-get update && \
+    apt-get install -y libboost-python-dev
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends python3.6-dev \
+    && ln -s -f /usr/bin/python3.6 /usr/bin/python \
+    && apt-get clean \
+    && rm -rf /var/cache/apt/archives/* /var/lib/apt/lists/*
+
+RUN curl -fSsL -O https://bootstrap.pypa.io/get-pip.py && \
+    python get-pip.py && \
```

Comment:
Are these needed? From my understanding we can just pip install required packages.